index.js

```diff
@@ -263,6 +267,7 @@ function SequestPut (conn, opts, path) {
   }
 
   this.connection = conn
+  this.connection.on('error', this.emit.bind(this, 'error'))
```

Comment:
I don't follow that.  Line 275 sets up a callback for when the connection transitions to ready for a new connection.

Are you think it should be in the else like this:

```javascript
  if (conn._state !== 'authenticated') {
   this.connection.on('error', this.emit.bind(this, 'error'))
    this.connection.on('ready', this.onConnectionReady.bind(this))
  } else {
    process.nextTick(function () {
      self.onConnectionReady()
    })
  }
```

Is that your thought?